src/Components/PaperEditor/ProgrammeScriptContainer/ExportDropdown/ExportModal/MediaModal.js

```diff
@@ -1,16 +1,60 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { useState } from 'react';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import {
   faDownload
 } from '@fortawesome/free-solid-svg-icons';
 
 import Modal from 'react-bootstrap/Modal';
 import Button from 'react-bootstrap/Button';
+import styles from './index.module.css';
```

Comment:
This is just a general preference for this project - the styling should be in-line where possible!